packages/terra-switch/src/terra-dev-site/doc/example/SwitchWithLongLabelText.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from 'react';
```

Comment:
This example doesn't need to be on the doc page. It's an edge case, so there should be a test for it, but consumers don't need to see this.

Reply:
example has been removed with this commit d8ea53ba069375ab4efb9abebf1bd67951aa067f